docs/features/multi_environment.md

```diff
@@ -443,13 +443,15 @@ Dev
 
     [feature.mlx]
     platforms = ["osx-arm64"]
+    # MLX is only available on macOS >=13.5 (>14.0 is recommended)
+    system-requirements = {macos = "13.5"}
 
     [feature.mlx.tasks]
     train-model = "python train.py --mlx"
     evaluate-model = "python test.py --mlx"
 
     [feature.mlx.dependencies]
-    mlx = ">=0.5.0,<0.6.0"
+    mlx = ">=0.15.0,<0.16.0"
```

Comment:
For me the latest version solves for 0.16.1, any reason not to go for that

Reply:
Not really, aside from this making a nicely symmetric diff. `">=0.16.0,<0.17.0"` should work fine - I'll amend it.

Reply:
Thanks, don't mind me using your PR to update the example too

Reply:
Of course not, thanks for the lightning-fast review!

Reply:
Thank you! I guess we're in the same timezone :wink: